src/UserRepository.js

```diff
@@ -16,6 +16,12 @@ class UserRepository {
     return Math.floor(totalUserSteps/this.data.length);
   }
 
+  getUsersByIds(userIds) {
+    const users = this.data.filter((user) => {
+      return userIds.includes(user.id);
+    });
+    return users;
+  }
```

Comment:
This method takes in an array of userIds and filters through the data to return user objects with those ids.